src/test/java/ru/training/at/hw1/classes/operations/arithmetic/AdditionTest.java

```diff
@@ -0,0 +1,16 @@
+package ru.training.at.hw1.classes.operations.arithmetic;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class AdditionTest {
+    @Test(groups = {"arithmeticOperations", "plusMinus"},
+            dataProvider = "operands", dataProviderClass = DataProvider.class)
+    public void testSum(int a, int b) {
+        Calculator testCalc = new Calculator();
```

Comment:
I suppose that you will create instance of the `Calculator` in each test. In this case creation of it could be extracted to the **base class** in `@before` and `@after` hooks

Reply:
I will extract the creation of all `Calculator` instances into a superclass and use the annotations